build.xml

```diff
@@ -26,7 +26,7 @@
   <property name="ts.home" value="${basedir}"/>
   <property name="deliverable.name" value="debugging-tck"/>
   <property name="eclipse.deliverable.name" value="jakarta-debugging-tck"/>
-  <property name="deliverable.version" value="2.0.0"/>
+  <property name="deliverable.version" value="2.0.1"/>
```

Comment:
+1 for catching this change!

Reply:
Actually, @RohitKumarJain @gurunrao did we make any changes for the DSOL tck to pass on JDK11?  If the 2.0.0 DSOL TCK already passes on JDK11, we don't need this change.

Reply:
> Actually, @RohitKumarJain @gurunrao did we make any changes for the DSOL tck to pass on JDK11? If the 2.0.0 DSOL TCK already passes on JDK11, we don't need this change.

revert complete.